internal/provider/resource_application_test.go

```diff
@@ -507,6 +507,33 @@ func TestAcc_ResourceApplication_UpdateEndpointBindings(t *testing.T) {
 	})
 }
 
+func TestAcc_ResourceApplication_Storage(t *testing.T) {
```

Comment:
This Test shouldn't run on microk8s, it doesn't support storage.

Reply:
fixed